docker-app/qfieldcloud/core/tests/test_qgis_file.py

```diff
@@ -56,6 +56,16 @@ def get_file_contents(self, project, filename, version=None):
         else:
             return response.content
 
+    def test_try_to_get_nonexistent_project(self):
+        empty_string = ""
+        nonexistent_id = "007"
+        with self.subTest(
+            "Ensure that '/api/v1/files' handles missing resources correctly."
+        ):
+            one = self.client.get(f"/api/v1/files/{empty_string}")
+            two = self.client.get(f"/api/v1/files/{nonexistent_id}")
+            assert all(r.status_code == 404 for r in (one, two))
```

Comment:
Just subjective little taste comment from myside: I would keep it simple such a test and rather just write `assertEqual(one.status_code, 404) etc` but is ok

Reply:
Forget about it, I will handle it.

Reply:
Thanks for the correction